wherescape/connectors/gitlab/gitlab_data_types_column_names.py

```diff
@@ -0,0 +1,143 @@
+"""
+COLUMN_NAMES_AND_DATA_TYPES is a dictionary with the flattened values and
+belonging data types returned from the Gitlab API
+"""
+
+COLUMN_NAMES_AND_DATA_TYPES = {
```

Comment:
Suggest to not only provide data types, but also column descriptions (descriptions column of [https://docs.gitlab.com/ee/api/projects.html#list-all-projects](https://docs.gitlab.com/ee/api/projects.html#list-all-projects) )